### PR TITLE
Cleanup Dragonfly embeds

### LINKED ISSUE #188: Make Dragonfly embeds consistent
Currently the embeds have different versions depending on which channels they're in.

### PR DESCRIPTION
**Pull request by shenanigansd**
Closes #188

### CODE REVIEW

File: src/bot/exts/dragonfly/dragonfly.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ async def report(self: Self, interaction: discord.Interaction, button: discord.u
 def _build_package_scan_result_embed(scan_result: PackageScanResult) -> discord.Embed:
     """Build the embed that shows the results of a package scan."""
     embed = discord.Embed(
-        title=f"New Scan Result: {scan_result.name} v{scan_result.version}",
+        title=f"Malicious package found: {scan_result.name} @ {scan_result.version}",
         description=f"```YARA rules matched: {', '.join(scan_result.rules) or 'None'}```",
```
Comment (shenanigansd) [inline src/bot/exts/dragonfly/dragonfly.py:135]: Is this capable of being `None` here?
If it's hit the threshold, it has to have rules, right?

File: src/bot/exts/dragonfly/dragonfly.py [hunk 2]
```diff
@@ -131,7 +131,7 @@ async def report(self: Self, interaction: discord.Interaction, button: discord.u
 def _build_package_scan_result_embed(scan_result: PackageScanResult) -> discord.Embed:
```
Comment (shenanigansd) [inline src/bot/exts/dragonfly/dragonfly.py:131]: This function name should be changed to something that actually specifies that it's a confirmed malicious result

File: src/bot/exts/dragonfly/dragonfly.py (hunk 1 above)
Comment (Robin5605) [reply src/bot/exts/dragonfly/dragonfly.py:135]: Yeah, but we can't really specify that in typing. Unless we have a separate type for that or something along those lines. Figured it'd just be easier to do this. But yeah, you're right in that we should never see this (unless the database was changed manually or something like that)

File: src/bot/exts/dragonfly/dragonfly.py (hunk 2 above)
Comment (Robin5605) [reply src/bot/exts/dragonfly/dragonfly.py:131]: `_build_malicious_package_scan_result_embed`? Feels a bit lengthy though

**Review by Robin5605 — APPROVED** [review]

**Review by Robin5605 — APPROVED** [review]

File: src/bot/exts/dragonfly/dragonfly.py (hunk 2 above)
Comment (shenanigansd) [reply src/bot/exts/dragonfly/dragonfly.py:131]: Hmm. I'll just leave this alone for now, and bring it up for discussion later.

**Pull request merged**